webpack/assets/javascripts/react_app/components/AutoComplete/auto-complete.scss

```diff
@@ -1,9 +1,11 @@
 @import "~@theforeman/vendor/scss/variables";
 
 .foreman-autocomplete {
+  position: relative;
+
   .autocomplete-aux {
     position: absolute;
-    right: 25px;
+    right: 4px;
```

Comment:
With positioning we can state what we want. - 4px from the right side.
:stop_sign: :man_cartwheeling: -> :walking_man: